### ISSUE TITLE
Chain support

### ISSUE THREAD

**Issue by lyledavids** [marc-aurele-besner/hardhat-awesome-cli#17]
Please add support for Optimism Mainnet and Optimism Kovan

**Comment by marc-aurele-besner** [comment]
Hi,
I have never used Optimistm yet 🤦‍♂️
Can you confirm me these values are correct?
```
{
        name: 'Optimistic Ethereum',
        chainName: 'optimistic',
        chainId: 10,
        gas: 'auto',
        defaultRpcUrl: 'hhttps://mainnet.optimism.io'
},
{
        name: 'Optimistic Ethereum Testnet Kovan',
        chainName: 'optimisticTestnetKovan',
        chainId: 56,
        gas: 'auto',
        defaultRpcUrl: 'https://bsc-dataseed1.binance.org'
}
```
Thank you

**Comment by lyledavids** [comment]
Hi, 
Thanks for the quick response.

Mainnet is correct.
For Optimism Kovan, the correct chainId is 69 and defaultRpcUrl is https://kovan.optimism.io/
Looks like the values you have is for BSC mainnet

**Comment by marc-aurele-besner** [comment]
Thank you :)
This has been included in #20 package v0.0.3 💪
